tests/integration/src/test/java/org/mule/properties/InvalidSetVariableTestCase.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.properties;
+
+import org.mule.api.MuleContext;
+import org.mule.api.config.ConfigurationException;
+import org.mule.config.spring.SpringXmlConfigurationBuilder;
+import org.mule.context.DefaultMuleContextFactory;
+import org.mule.tck.junit4.AbstractMuleTestCase;
+
+import org.junit.Test;
+
+public class InvalidSetVariableTestCase extends AbstractMuleTestCase
```

Comment:
this test is exactly the same as the one before. Have only one which uses @Parametrized to test each scenario